validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala

```diff
@@ -6,19 +6,19 @@ import hmda.validation.context.ValidationContext
 import hmda.validation.engine.Syntactical
 import hmda.validation.engine.lar.LarCommonEngine
 import hmda.validation.rules.EditCheck
-import hmda.validation.rules.ts.syntactical.S025
+import hmda.validation.rules.ts.syntactical.{ S020, S025 }
 import hmda.validation.rules.lar.syntactical._
 
 trait LarSyntacticalEngine extends LarCommonEngine with ValidationApi {
 
   def checkSyntactical(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
-    val checksToRun: List[EditCheck[LoanApplicationRegister]] = List(
+    val checksToRun: List[EditCheck[_ >: LoanApplicationRegister]] = List(
       S010,
       S020,
       S025.inContext(ctx),
       S205
     )
-    val checks = checksToRun.map(check(_, lar, lar.loan.id, Syntactical))
+    val checks = checksToRun.map(check[LoanApplicationRegister](_, lar, lar.loan.id, Syntactical))
```

Comment:
With the code as it is right now, removing that type leads to a compile error.
You're right that `TsSyntacticalEngine` seems not to need it, though, even though it has similar types involved. I wonder what the relevant difference is! Definitely worth exploring.